src/main/java/uk/gov/hmcts/reform/sscs/model/hmc/reference/HmcStatus.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.sscs.model.hmc.reference;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.HEARING_BOOKED;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.UPDATE_CASE_ONLY;
+
+@RequiredArgsConstructor
+@Getter
+public enum HmcStatus {
```

Comment:
Q: I'm guessing these null and empty values will be populated at some point?

Reply:
I presume so, we just don't handle these at the moment